swift/ql/lib/codeql/swift/frameworks/StandardLibrary/Url.qll

```diff
@@ -22,7 +22,41 @@ private class UrlSummaries extends SummaryModelCsv {
+        ";URL;true;init(string:);(String);;Argument[0];ReturnValue.Field[pathExtension];taint",
+        ";URL;true;init(string:);(String);;Argument[0];ReturnValue.Field[port];taint",
+        ";URL;true;init(string:);(String);;Argument[0];ReturnValue.Field[query];taint",
+        ";URL;true;init(string:);(String);;Argument[0];ReturnValue.Field[relativePath];taint",
+        ";URL;true;init(string:);(String);;Argument[0];ReturnValue.Field[relativeString];taint",
+        ";URL;true;init(string:);(String);;Argument[0];ReturnValue.Field[scheme];taint",
+        ";URL;true;init(string:);(String);;Argument[0];ReturnValue.Field[standardized];taint",
+        ";URL;true;init(string:);(String);;Argument[0];ReturnValue.Field[standardizedFileURL];taint",
+        ";URL;true;init(string:);(String);;Argument[0];ReturnValue.Field[user];taint",
+        ";URL;true;init(string:);(String);;Argument[0];ReturnValue.Field[password];taint",
+        ";URL;true;init(string:relativeTo:);(String,URL?);;Argument[0,1];ReturnValue;taint",
+        ";URL;true;init(string:relativeTo:);(String,URL?);;Argument[0,1];ReturnValue.Field[absoluteURL];taint",
+        ";URL;true;init(string:relativeTo:);(String,URL?);;Argument[0,1];ReturnValue.Field[baseURL];taint",
+        ";URL;true;init(string:relativeTo:);(String,URL?);;Argument[0,1];ReturnValue.Field[fragment];taint",
+        ";URL;true;init(string:relativeTo:);(String,URL?);;Argument[0,1];ReturnValue.Field[host];taint",
+        ";URL;true;init(string:relativeTo:);(String,URL?);;Argument[0,1];ReturnValue.Field[lastPathComponent];taint",
+        ";URL;true;init(string:relativeTo:);(String,URL?);;Argument[0,1];ReturnValue.Field[path];taint",
+        ";URL;true;init(string:relativeTo:);(String,URL?);;Argument[0,1];ReturnValue.Field[pathComponents];taint",
+        ";URL;true;init(string:relativeTo:);(String,URL?);;Argument[0,1];ReturnValue.Field[pathExtension];taint",
+        ";URL;true;init(string:relativeTo:);(String,URL?);;Argument[0,1];ReturnValue.Field[port];taint",
+        ";URL;true;init(string:relativeTo:);(String,URL?);;Argument[0,1];ReturnValue.Field[query];taint",
+        ";URL;true;init(string:relativeTo:);(String,URL?);;Argument[0,1];ReturnValue.Field[relativePath];taint",
+        ";URL;true;init(string:relativeTo:);(String,URL?);;Argument[0,1];ReturnValue.Field[relativeString];taint",
+        ";URL;true;init(string:relativeTo:);(String,URL?);;Argument[0,1];ReturnValue.Field[scheme];taint",
+        ";URL;true;init(string:relativeTo:);(String,URL?);;Argument[0,1];ReturnValue.Field[standardized];taint",
+        ";URL;true;init(string:relativeTo:);(String,URL?);;Argument[0,1];ReturnValue.Field[standardizedFileURL];taint",
+        ";URL;true;init(string:relativeTo:);(String,URL?);;Argument[0,1];ReturnValue.Field[user];taint",
+        ";URL;true;init(string:relativeTo:);(String,URL?);;Argument[0,1];ReturnValue.Field[password];taint",
```

Comment:
It would be good to have a couple of comments explaining what blocks of these rows do. 
 Something like your PR description:
```
If a URL is tainted, so are its fields.
```

Reply:
Good suggestion, applied in 1e48500, thanks!